packages/central-server/src/apiV2/meditrakApp/meditrakSync/meditrakSyncQuery.js

```diff
@@ -4,13 +4,12 @@
  */
 
 import { get } from 'lodash';
-import semverCompare from 'semver-compare';
-
+import { compare } from 'compare-versions';
 import { getHighestPossibleIdForGivenTime, SqlQuery } from '@tupaia/database';
 import { ValidationError } from '@tupaia/utils';
 import { fetchRequestingMeditrakDevice } from '../utilities';
 
-const isAppVersionGreaterThanMin = (version, minVersion) => semverCompare(version, minVersion) >= 0;
+const isAppVersionGreaterThanMin = (version, minVersion) => compare(version, minVersion, '>=');
```

Comment:
Changed package for comparing, as 'compare-versions' is much more up to date and also human readable.

Reply:
Nice 👍